tests/jobs/test_job_decorator.py

```diff
@@ -489,9 +496,12 @@ async def execute(self) -> bool:
     coresys.updater.auto_update = False
     assert not await test.execute()
 
+    coresys.jobs.ignore_conditions = [JobCondition.AUTO_UPDATE]
+    assert await test.execute()
 
-async def test_unhealthy(coresys: CoreSys, caplog: pytest.LogCaptureFixture):
```

Comment:
This test was basically a duplicate of the `test_healthy` test. Moved the log check bit into `test_healthy` and removed this.